### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by drpdr**
The README.md is old and it needed a refresh.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,49 @@
...
 
-The last step would be to execute the split utility. This can be done by 
-running it like that: 
 ```
-./split-llvm-extract python.bc -o outdir
+./split-llvm-extract path/to/binary.bc -o outdir
 ```
 
-## Other variants
+To run the version of the binary just split, it is necessary to join the parts (in `outdir`) together as shared libraries.
```
Comment (ltratt) [inline README.md:36]: Let's wrap these lines to the same length as the others in this file.

Comment (drpdr) [reply README.md:36]: Fix in [2fe2b9e](https://github.com/capablevms/llvm-function-split/pull/37/commits/2fe2b9e95ef3df217b2dbd2c87f813e8bf6946dd), I *think".

Comment (ltratt) [reply README.md:36]: Nope, that's made it much worse ;) That commit has totally unwrapped a number of lines.

It looks to me (at a quick glance) that this file, in master, has lines wrapped to 80ish chars. Let's just keep things the same, whatever that is.

Comment (drpdr) [reply README.md]: What about [78da4f5](https://github.com/capablevms/llvm-function-split/pull/37/commits/78da4f5812dab9d8c692a50cd46e21c00130bae9)?

Comment (ltratt) [reply README.md]: OK, but that's reformatted the whole file ;) What I'm really asking is: revert back to https://github.com/capablevms/llvm-function-split/pull/37/commits/085d18545c222b67a5fbdbef4b9824fa44f98869 then wrap only those lines/paragraphs you changed in that commit.

Comment (drpdr) [reply README.md]: Check [57183c8](https://github.com/capablevms/llvm-function-split/pull/37/commits/57183c826455ce015458d5f958ae3fa9c98bef31), please.

**Comment by drpdr** [comment]
Is this OK, @ltratt ?

**Comment by ltratt** [comment]
Please squash.

**Comment by drpdr** [comment]
> Please squash.

Done.

**Pull request merged**